go.mod

```diff
@@ -132,7 +131,9 @@ require (
 	github.com/google/pprof v0.0.0-20240727154555-813a5fbdbec8 // indirect
 	github.com/google/s2a-go v0.1.9 // indirect
 	github.com/hashicorp/errwrap v1.1.0 // indirect
+	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
```

Comment:
looks like these are still indirects (probably because of us supporting hashivault)... is that still an issue for CNCF?

Reply:
I think so, but the code does not seem to be imported from kubernetes-sigs/release-sdk, we may follow-up on that, though.